src/modules/iotjs_module_buffer.c

```diff
@@ -211,47 +198,42 @@ jerry_value_t iotjs_bufferwrap_create_buffer(size_t len) {
 
 
 JS_FUNCTION(Buffer) {
-  DJS_CHECK_THIS();
   DJS_CHECK_ARGS(2, object, number);
 
-  const jerry_value_t jbuiltin = JS_GET_THIS();
-  const jerry_value_t jbuffer = JS_GET_ARG(0, object);
+  const jerry_value_t jbuiltin = JS_GET_ARG(0, object);
```

Comment:
I suggest naming `jobject` or `jcontext` instead of `jbuiltin`. `iotjs_bufferwrap_create` also needs the same.